data/birth-dates/10.txt

```diff
@@ -1 +1 @@
-Edgar F. Codd: 23. August ????
```

Comment:
Why do you think this should be 23rd?

Reply:
https://en.wikipedia.org/wiki/Edgar_F._Codd It is the 19th according to wikipedia

Reply:
ok